xla/service/gpu/cudnn_support_utils.cc

```diff
@@ -48,13 +49,21 @@ absl::StatusOr<bool> CudnnSupportsOptimizedIntegerConvolution(
     return false;
   }
 
-  // Require cc6.1+ for any vectorized integer convolutions
-  // Require cc7.5+ for any IMMA convolutions
-  if ((vector_size == 32 && !compute_capability.IsAtLeast(7, 5)) ||
-      !compute_capability.IsAtLeast(6, 1)) {
-    VLOG(3) << "Compute capability " << compute_capability.ToString()
-            << " is not sufficent for int8x" << vector_size
-            << " vectorization.";
+  bool disabled = std::visit(VariantVisitor{
+      [vector_size](const se::CudaComputeCapability& cc) {
```

Comment:
`[&]` capture is enough here, no async dispatch happens here, no need to capture things by value